documentation/docs/30-advanced/10-advanced-routing.md

```diff
@@ -123,27 +123,33 @@ src/routes/[...catchall]/+page.svelte
 
 ### Encoding
 
-Directory names are URI-decoded, meaning that (for example) a directory like `%40[username]` would match characters beginning with `@`:
+Some characters can't be used on the filesystem — `/` on Linux and Mac, `\ / : * ? " < > |` on Windows. The `#` character has special meaning in URLs, and the `[ ] ( )` characters have special meaning to SvelteKit, so these also can't be used directly as part of your route.
 
-```js
-// @filename: ambient.d.ts
-declare global {
-	const assert: {
-		equal: (a: any, b: any) => boolean;
-	};
-}
+To use these characters in your routes, you can use hexadecimal escape sequences:
 
-export {};
+- `\` — `[x+5c]`
+- `/` — `[x+2f]`
+- `:` — `[x+3a]`
+- `*` — `[x+2a]`
+- `?` — `[x+3f]`
+- `"` — `[x+22]`
+- `<` — `[x+3c]`
+- `>` — `[x+3e]`
+- `|` — `[x+7c]`
+- `#` — `[x+23]`
+- `[` — `[x+5b]`
+- `]` — `[x+5d]`
+- `(` — `[x+28]`
+- `)` — `[x+29]`
 
-// @filename: index.js
-// ---cut---
-assert.equal(
-	decodeURIComponent('%40[username]'),
-	'@[username]'
-);
-```
+For example, to create a `/:-)` route, you would create a `src/routes/[x+3a]-[x+29]/+page.svelte` file.
```

Comment:
I think there are four character in the non-escaped version and three in the escaped version, so one is probably missing

Reply:
I don't follow? The path segment is `:-)`, which is represented by `[x+3a]-[x+29]`

Reply:
oh, I thought `/` was part of the route ID (I think it was in the last PR). Maybe we could change it then:
```suggestion
For example, to create a `:-)` route, you would create a `src/routes/[x+3a]-[x+29]/+page.svelte` file.
```

Reply:
`/` _is_ part of the route ID — all route IDs are prefixed with `/`, we're fairly consistent about that 
<img width="753" alt="image" src="https://user-images.githubusercontent.com/1162160/201756062-35aade7b-84e9-4b6e-97ef-8d196039fde7.png">

maybe if it was `/smileys/:-)` or `/emoticons/:-)`?

Reply:
I changed it to `/smileys/:-)`